src/util/tracking.ts

```diff
@@ -48,22 +50,23 @@ export type TrackingEventName =
   | 'Earn_Spend_Launch'
   | LoginTrackingEventName
 
+export type OnLogEvent = (event: TrackingEventName, values?: TrackingValues) => void
+
 export interface TrackingValues extends LoginTrackingValues {
-  accountDate?: string // Account creation date
   currencyCode?: string // Wallet currency code
   dollarValue?: number // Conversion amount, in USD
   error?: unknown | string // Any error
-  installerId?: string // Account installerId, i.e. referralId
   orderId?: string // Unique order identifier provided by plugin
   pluginId?: string // Plugin that provided the conversion
   numSelectedWallets?: number // Number of wallets to be created
   destCurrencyCode?: string
-  destExchangeAmount?: string
+  destExchangeAmount?: number
```

Comment:
why did you change this to a number? This just added more work in the PR for everywhere this is passed into logEvent.

Reply:
This was changed because we need to make sure we're always reporting numeric values to PostHog so we can properly filter gt, Lt, etc

Reply:
Changed back to a string and just did a check within `logEvent` to ensure the string represents a number. 

This is very important because if we pollute a value field with a string that can't be interpreted as a number, we lose numeric comparison functionality for that field in PostHog.

Reply:
I fully support using numbers as numbers. We should only be using biggystring in cases where we need absolute correctness (like sending customer funds), and we are willing to pay the price in terms of performance and inconvenience. Otherwise switching to plain numbers is a helpful cleanup.

Reply:
While I mostly agree, the risk of having a conversion that leaves us with an invalid, non-parsable string (scientific notation) is risky. At minimum our nomenclature should be consistent. `nativeAmount` and `exchangeAmount` are used extensively throughout the app and they are always strings. The introduction of nativeAmount in this PR https://github.com/EdgeApp/edge-react-gui/pull/4825 actually introduces a bug due to blindly converting `nativeAmount` to a number and assuming it's usable.

Reply:
That makes sense.

Reply:
We do stringify numbers coming from our partners in the plugins sometimes, but I'm opting to keep it consistent with string types only in our internal-facing API for consistency.